templates/source_template.hjson

```diff
@@ -61,7 +61,8 @@
                     OPTIONAL { ?id rdfs:label ?label_en . FILTER(LANG(?label_en) = "en") }
               }
               # remove possible ID duplicates
-              GROUP BY ?id}
+              GROUP BY ?id
+              ORDER BY ?id}
```

Comment:
would enforcing this ordering have any unexpected effects? (just asking, probably not, but maybe missing something her)

Reply:
I don't think so. I still do some post-processing on the data after retrieval to sort the features for better git diffing. So this isn't necessary here. I'll remove it.